packages/ui/src/modal/acceptBid/AcceptBidModal.tsx

```diff
@@ -177,8 +182,11 @@ export function AcceptBidModal({
           [tokensData]
         )
 
+        const saleStep = stepData?.steps.find((step) => step.id === 'sale')
+        const swapStep = stepData?.steps.find((step) => step.id === 'swap')
+
         const transfersTxHashes =
-          stepData?.currentStep?.items?.reduce((txHashes, item) => {
+          saleStep?.items?.reduce((txHashes, item) => {
             item.transfersData?.forEach((transferData) => {
```

Comment:
We use the saleStep here since this is the step that will always have the final TXes.

Reply:
should we group up all transactions to display to users?